packages/fast-components-react-base/src/hypertext/hypertext.props.ts

```diff
@@ -0,0 +1,14 @@
+import * as React from "react";
+import {IHypertextClassNameContract, IManagedClasses} from "@microsoft/fast-components-class-name-contracts";
+
+export interface IHypertextHandledProps {
+
+    href?: string;
+
+    children?: React.ReactNode | React.ReactNode[];
+}
+
+
+export interface IHypertextUnhandledProps extends React.AllHTMLAttributes<HTMLElement> {}
```

Comment:
I don't think we want all html attributes here as this is just an anchor. We likely want:
`React.HTMLAnchorAttributes<HTMLAnchorElement>`